java/arcs/core/util/Analytics.kt

```diff
@@ -0,0 +1,62 @@
+/*
+ * Copyright 2020 Google LLC.
+ *
+ * This code may only be used under the BSD style license found at
+ * http://polymer.github.io/LICENSE.txt
+ *
+ * Code distributed by Google as part of this project is also subject to an additional IP rights
+ * grant found at
+ * http://polymer.github.io/PATENTS.txt
+ */
+package arcs.core.util
+
+import arcs.core.util.Log.formatter
+import arcs.core.util.Log.writer
+
+/**
+ * Analytics entrance for Arcs.
+ *
+ * Allows for pluggable output sinks (see [logger]).
+ */
+object Analytics {
```

Comment:
Would it make sense to have `Analytics` implement `Logger`? and have its implementation just proxy through to the actual instance stored in `var logger`?  This might feel cleaner unless you anticipate more fields, with things different from the `logger` interface exposed as well.